HW04/Vladislav_Mitin/lib/mentor.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Mentor
+  attr_reader :name, :surname, :notifications, :subscriptions
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @subscriptions = []
+    @notifications = []
+  end
+
+  def add_homework(title:, description:, student:)
+    homework = Homework.new(title: title, description: description, student: student, mentor: self)
+    notification = Notification.new(homework: homework)
+
+    student.homeworks << homework
+    student.notifications << notification
```

Comment:
Try to avoid object modification outside the class. Would be better to create a new interface and wrap it.
```suggestion
    student.notify(notification)
    
    # inside the student class
    def notify(notification)
       notifications << notification
    end
```